src/api/app/controllers/webui/project_controller.rb

```diff
@@ -85,6 +85,8 @@ def users
   def subprojects
     @subprojects = @project.subprojects.order(:name)
     @parentprojects = @project.ancestors.order(:name)
+    @parent = @project.parent
```

Comment:
You only use the parent name in the view, right? why not  `@parent_name = @project.parent.name` ?